app/controllers/custom_styles_controller.rb

```diff
@@ -44,10 +44,16 @@ class CustomStylesController < ApplicationController
                      only: UNGUARDED_ACTIONS
   no_authorization_required! *UNGUARDED_ACTIONS
 
+  def default_url_options
+    super.merge(tab: params[:tab])
+  end
   def show
     @custom_style = CustomStyle.current || CustomStyle.new
     @current_theme = @custom_style.theme
     @theme_options = options_for_theme_select
+    if params[:tab].blank?
```

Comment:
Nitpick: Consider adding an empty line before the if-statement to give the condition-part of the method a visual distinction to the assignment-part of the method.

I know that this controller is rather crammed in the other methods, but it would really improve the readability if we stepped up our formatting here.